modules/bloxone_infra_host_aws/main.tf

```diff
@@ -54,7 +54,7 @@ data "aws_ami" "bloxone" {
   most_recent = true
   filter {
     name   = "name"
-    values = ["BloxOne381_MarketPlace-96cf85a8-a940-4dd0-80a5-80ab90fb1d1a"]
+    values = ["Infoblox BloxOne 3.8.1 BYOL"]
```

Comment:
this image is not in the marketplace. it seems the marketplace image was updated and renamed.

Reply:
Fixed